blocks/library/block/edit-panel/index.js

```diff
@@ -3,65 +3,80 @@
  */
 import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { keycodes } from '@wordpress/utils';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
+const { ESCAPE } = keycodes;
+
 function ReusableBlockEditPanel( props ) {
 	const { isEditing, name, isSaving, onEdit, onDetach, onChangeName, onSave, onCancel } = props;
 
-	return (
-		<div className="reusable-block-edit-panel">
-			{ ! isEditing && ! isSaving && [
-				<span key="info" className="reusable-block-edit-panel__info">
-					<b>{ name }</b>
-				</span>,
-				<Button
-					key="edit"
-					isLarge
-					className="reusable-block-edit-panel__button"
-					onClick={ onEdit }>
-					{ __( 'Edit' ) }
-				</Button>,
-				<Button
-					key="detach"
-					isLarge
-					className="reusable-block-edit-panel__button"
-					onClick={ onDetach }>
-					{ __( 'Detach' ) }
-				</Button>,
-			] }
-			{ ( isEditing || isSaving ) && [
+	return [
+		! isEditing && ! isSaving && <div className="reusable-block-edit-panel">
+			<span key="info" className="reusable-block-edit-panel__info">
+				<b>{ name }</b>
+			</span>
+			<Button
+				key="edit"
+				isLarge
+				className="reusable-block-edit-panel__button"
+				onClick={ onEdit }>
+				{ __( 'Edit' ) }
+			</Button>
+			<Button
+				key="detach"
+				isLarge
+				className="reusable-block-edit-panel__button"
+				onClick={ onDetach }>
+				{ __( 'Detach' ) }
+			</Button>
+		</div>,
+		( isEditing || isSaving ) && (
+			<form
+				className="reusable-block-edit-panel"
```

Comment:
This element needs a `key`, and conversely, its children no longer need `key`s.

>react-dom.d3299c8d.js:894 Warning: Each child in an array or iterator should have a unique "key" prop. See https://fb.me/react-warning-keys for more information.

Reply:
👌 Fixed in 8946b8d6fd1716e06ac30cd4ed4d137ff8889dcb